### ISSUE TITLE
Test microservice without creating an app

### ISSUE THREAD

**Issue by vsDizzy** [nestjs/nest#2350]
I have a testing module `mod` and my controller handlers are not registered if i don't use an app.

This works
```ts
  const app = mod.createNestApplication();
  app.connectMicroservice({ strategy: sv });
  console.log(sv.getHandlers().size);
```
1

This is not working
```ts
  mod.createNestMicroservice({ strategy: sv });
  console.log(sv.getHandlers().size);
```
0

Maybe I'm missing something. Creating an app is an overkill.
Can we just use microservice only?
Please help.

**Comment by kamilmysliwiec** [comment]
You have to initialize (`await mod.init()`) your application (or just call `listen()`).

**Comment by vsDizzy** [comment]
`await mod.init()` does not work, however `listen()` does.
Can we make `init` work without listen please?

**Comment by kamilmysliwiec** [comment]
It should be fixed in 6.4.1 :)
